main.go

```diff
@@ -1,14 +1,20 @@
 package main
 
-import "fmt"
-import "github.com/A9u/altilia/internal/device"
+import (
+	"fmt"
+
+	"github.com/A9u/altilia/internal/device"
+)
 
 func main() {
 	fmt.Println("----------Welcome to Altilia----------")
 	fmt.Println()
 	fmt.Println("Checking Battery status")
-	fmt.Println(device.GetPower())
-	fmt.Println()
-	fmt.Println("Notifying")
-	device.Notify()
+	fmt.Println("")
+	device.CheckPower()
+	go device.AmICharged()
+
+	waitChannel := make(chan string)
+
+	<-waitChannel
```

Comment:
This channel receive statement doesn't seem like it can ever complete, does it? Who will send to `waitChannel`?

Reply:
This channel is used to keep the main thread running 😛 .  Nobody will send anything to waitChannel